nixos/modules/services/games/minecraft-server.nix

```diff
@@ -158,6 +158,32 @@ in {
           + "-XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10";
         description = "JVM options for the Minecraft server.";
       };
+
+      fifo = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "/run/mincraft/stdin";
+        description = "create a FiFo device to send commands to the minecraft process";
```

Comment:
How about
```nix
''
  The path to a FIFO for sending commands to standard input of the
  Minecraft server process. E.g. this can be used to op players with
  <command>echo "op Steve" > ''${standardInputFifo}</command>.
''
```